src/views/ModelPlan/TaskList/Payment/Complexity/index.tsx

```diff
@@ -0,0 +1,544 @@
+                                as={TextField}
+                                error={
+                                  flatErrors.canParticipantsSelectBetweenPaymentMechanismsHow
+                                }
+                                id="payment-multiple-payments-how"
+                                data-testid="payment-multiple-payments-how"
+                                name="canParticipantsSelectBetweenPaymentMechanismsHow"
+                              />
+                            </FieldGroup>
+                          )}
+                          <Field
+                            as={Radio}
+                            id="payment-multiple-payments-No"
+                            name="payment-multiple-payments"
+                            label={h('no')}
+                            value="NO"
+                            checked={
+                              values.canParticipantsSelectBetweenPaymentMechanisms ===
+                              false
+                            }
+                            onChange={() => {
+                              setFieldValue(
+                                'canParticipantsSelectBetweenPaymentMechanisms',
+                                false
+                              );
+                            }}
+                          />
+                        </Fieldset>
+                        <AddNote
+                          id="payment-multiple-payments-note"
+                          field="canParticipantsSelectBetweenPaymentMechanismsNote"
+                        />
+                      </FieldGroup>
+
+                      <FieldGroup
+                        scrollElement="payment-frequency-payments"
+                        error={!!flatErrors.anticipatedPaymentFrequency}
+                        className="margin-top-4"
+                      >
+                        <Label htmlFor="payment-frequency-payments">
+                          {t('anticipatedPaymentFrequency')}
+                        </Label>
+                        <FieldErrorMsg>
+                          {flatErrors.anticipatedPaymentFrequency}
+                        </FieldErrorMsg>
+
+                        <Field
+                          as={MultiSelect}
+                          id="payment-frequency-payments"
+                          name="payment-frequency-payments"
+                          options={mappedAnticipatedPaymentFrequencyType}
+                          selectedLabel={t(
+                            'selectedAnticipatedPaymentFrequency'
+                          )}
+                          onChange={(value: string[] | []) => {
+                            setFieldValue('anticipatedPaymentFrequency', value);
+                          }}
+                          initialValues={
+                            initialValues.anticipatedPaymentFrequency
+                          }
+                        />
```

Comment:
The enum values for this field are also throwing gql errors.  May need to consult with BE for this as well.  Not sure what's up on first glance.